website/blog/releases/3.0/index.mdx

```diff
@@ -0,0 +1,384 @@
+---
+title: Announcing Docusaurus 3.0
+authors:
+  - slorber
+tags: [release]
+image: ./img/social-card.png
+date: 2023-10-25
+unlisted: true
+---
+
+Today we are happy to **announce Docusaurus 3.0**! 🥳
+
+At [**Meta Open Source**](https://opensource.fb.com/), we believe Docusaurus will help you build the **best documentation websites** with **minimal effort**, letting you **focus on what really matters**: writing the content.
+
+This is a new **major version** of Docusaurus, coming with **new exciting features** and upgraded dependencies.
+
+We respect [**Semantic Versioning**](https://semver.org/) and this release also includes **breaking changes** that we documented thoroughly in the [**v3 upgrade guide**](/docs/3.0.0-rc.0/migration/v3). Breaking changes can be annoying, but are necessary to set the ground for a **new wave of Docusaurus features** that we plan to implement later.
```

Comment:
In line with the [Semantic Versioning](https://semver.org/) principles, this release includes breaking changes we documented thoroughly in the [v3 upgrade guide](https://deploy-preview-9433--docusaurus-2.netlify.app/docs/3.0.0-rc.0/migration/v3/). Breaking changes can be bothersome, but they are necessary to set the ground for a new wave of Docusaurus features we plan to implement.